app/src/main/java/io/apicurio/registry/mt/TenantContextLoader.java

```diff
@@ -75,13 +75,23 @@ public void onStart(@Observes StartupEvent ev) {
         contextsCache = new CheckPeriodCache<>(cacheCheckPeriod);
     }
 
-    public RegistryTenantContext loadContext(String tenantId) {
+    public RegistryTenantContext loadRequestContext(String tenantId) {
+        return loadContext(tenantId, true);
+    }
+
+    public RegistryTenantContext loadBatchJobContext(String tenantId) {
+        return loadContext(tenantId, false);
+    }
+
+    private RegistryTenantContext loadContext(String tenantId, boolean checkTenantAuthorization) {
```

Comment:
I would add a comment here explaining a bit why the tenant authorization can be skipped. It seems obvious right now but it might be no that obvious in e.g. 1 year.

Reply:
done, thank you for the reminder